pkg/image/registry/image/strategy.go

```diff
@@ -38,8 +38,8 @@ func (s imageStrategy) PrepareForCreate(obj runtime.Object) {
 	if err := api.ImageWithMetadata(newImage); err != nil {
 		utilruntime.HandleError(fmt.Errorf("Unable to update image metadata for %q: %v", newImage.Name, err))
 	}
-
-	s.clearSignatureDetails(newImage)
+	// allow to set the signatures only through signatures subresource
```

Comment:
> Repasting @liggitt's earlier comment here:
> 
> I'm not sure whether I'd expect the first person to push an image to be able to surface their signatures into any other namespace they want. @smarterclayton, was that what you expected?

The cluster-admin is the cluster-admin.  If he wants the signatures, he can have them.

If we want to argue about expectations during create and stripping the field during create as a general rule for usability, I could probably see that argument, but that means that updates can still apply them and we can still loopback through the API server to make it work.